app/views/catalog/_svccat_tree_show.html.haml

```diff
@@ -28,6 +28,12 @@
           .col-md-8
             %p.form-control-static#long_description
               %miq-sanitize{:value => @record.long_description}
+        .form-group
+          %label.col-md-2.control-label
+            = _('Dialog')
+          .col-md-8
+            %p.form-control-static
+              = @sb[:dialog_label]
```

Comment:
Should this be `= h(@sb[:dialog_label])` as in sand_tree_show and ab_show?

Reply:
..never mind :)

Looks like that's just an erb artifact..

```haml
- foo = 'ba<b>gr</b>ooo<br />'

Foo
= foo

Bar
= h(foo)
```

the output for both is exactly the same